pep/app/constants/search.ts

```diff
@@ -243,6 +258,8 @@ export const SEARCH_TYPE_ARTICLE: SearchTermType = {
     param: 'fulltext1',
     solrField: 'body_xml',
     scope: 'doc',
+    wordWheelField: 'body_xml',
+    wordWheelCore: 'docs',
```

Comment:
Yeah it can, I'm just not sure yet if any of these fields need it changed for their queries, so I was gonna let Neil play with it and adjust based on his feedback. but we can make this a const or just remove it here altogether after we get that feedback if it doesnt change (or make it an enum if it does change).

Reply:
FYI - There are three cores currently.  Docs, Glossary, and Authors.  Most of the endpoints go to Docs, but not all.  Glossary goes to both Docs and Glossary, since the glossary documents are just normal docs, but indexed at a different level in the Glossary core.  

In other words, word wheels may be useful for each core.  Which core is searched by an endpoint depends on the level of the solr document "entity", e.g., each "document" in the Solr database is an article in the Docs core, a Glossary group or term in the Glossary core, and an Author in the authors core.  (There was also a References core but for now anyway it's been deprecated and the SQL database is used for those instead...it was better for crosstabs.)

Reply:
Thanks @nrshapiro. 

So ultimately, what core would you like each word wheel to be set to in the search form fields specifically (i.e. the Smart Search input, and each of the fielded search term type inputs, i.e. Everywhere, Author, Title, etc.)?

I currently have them all set to "docs", but I did try using the "authors" core for the 'Author' term field, but I didn't seem to be getting any results with that. For example, this returns no results on staging:

```
GET /v2/Database/WordWheel/?core=authors&field=art_authors&limit=8&offset=0&word=smith
```

While this request does return results that appear to be correct:
```
GET /v2/Database/WordWheel/?core=docs&field=art_authors&limit=8&offset=0&word=smith
```